### PR TITLE
Fix for box-shadow/border-image issues on blockquotes in new columns

### PR DESCRIPTION
**Pull request by G-Ambatte**
As per issue reported on Reddit [here](https://redd.it/kpcbw6), with the fix found by /u/alexey152.

I was able to replicate the issue in Chrome and Edge, but not Firefox.  
I have confirmed that this change fixes the issue in both Chrome and Edge.

### CODE REVIEW

**Comment by calculuschild** [comment]
Wow, weird solution... Can you verify if this shifts the elements around on the page at all? If not, we can apply this fix, but if it shifts boxes around it might ruin some old brews, and in that case we can save this for the new renderer.

**Comment by G-Ambatte** [comment]
Doesn't look like it shifts the elements at all - here's some screengrabs:

[Before](https://i.imgur.com/unmHP47.png) || [After](https://imgur.com/eM8rq1R)  
[Before](https://imgur.com/LK8FyCz) || [After](https://imgur.com/eSc6utm)

My testing so far has shown that `transform: rotateZ(360);` doesn't resolve the issue. [This](https://homebrewery.naturalcrit.com/share/1jumEkGO1ICHEu0rLSaZqwNvVhTe5495NlkoXck9lLS4P) is the share link to my reproduction of the issue.

**Comment by calculuschild** [comment]
Huh. Well alright then. Doesn't look like it will be a breaking change based on your screenshots. @AlexeySachkov, @ericscheid , any concerns merging this?

**Comment by ericscheid** [comment]
Looks good to me - the modded result only shifts the shadow to where it should be, it doesn't bump the position of the box being shadowed.

**Review by ericscheid — APPROVED** [review]

**Comment by AlexeySachkov** [comment]
I found this solution in [this](https://stackoverflow.com/questions/17881923/box-shadow-trimmed-in-css-columns-in-chrome) stackoverflow question and I believe that this is a bug in Chrome and we should probably put a `TODO`/`FIXME` comment in the code to remove it once Chrome is fixed. [This](https://bugs.chromium.org/p/chromium/issues/detail?id=269061) seems to be a bug report in Chromium project

**Comment by Gazook89** [comment]
I'm on phone so can't be sure, but this is also related to issue #610? In which another solution is offered (use flexbox, not columns) but imagine that would create many more issues. But if this PR goes through, that issue can be closed.

File: client/homebrew/phbStyle/phb.style.less [hunk 1]
```diff
@@ -189,6 +189,7 @@ body {
 		border-image        : @noteBorderImage 11;
 		border-image-outset : 9px 0px;
 		box-shadow          : 1px 4px 14px #888;
+		transform           : translateZ(0);
```
Comment (Gazook89) [inline client/homebrew/phbStyle/phb.style.less:192]: [Per this comment on the stackoverflow solution,](https://stackoverflow.com/questions/17881923/box-shadow-trimmed-in-css-columns-in-chrome#comment103838351_55110789) perhaps better to prefix with -webkit- so as to only kick in for Chrome users?

Comment (AlexeySachkov) [reply client/homebrew/phbStyle/phb.style.less:192]: We have a "Render Warning", that says that Homebrewery is built & tested for Chrome mostly, so I would expect the majority of users to use it and therefore, it doesn't seem like a critical fix to have; but yes, we can apply this change, I have no objections

**Comment by AlexeySachkov** [comment]
> I'm on phone so can't be sure, but this is also related to issue #610? 
> Edit : and #620 probably

Yeah, those seem to be duplicates. I suggest that we close them and refer to this PR

> In which another solution is offered (use flexbox, not columns) but imagine that would create many more issues. But if this PR goes through, that issue can be closed.

Using "flexbox" might be a solution for this issue, but I wouldn't consider it as a quick fix, because most likely it will cause some rendering changes that will affect existing brews. We can for sure explore this way to see if it is easier to manage for users, maintain for developers and whether it is more reliable or flexible, but it should probably be done as a separate activity and not in scope of a bugfix.

**Comment by Gazook89** [comment]
Yeah, the first thing I think about with flexbox is how to replace column spanning/wide blocks.  

> Using "flexbox" might be a solution for this issue, but I wouldn't consider it as a quick fix, because most likely it will cause some rendering changes that will affect existing brews. We can for sure explore this way to see if it is easier to manage for users, maintain for developers and whether it is more reliable or flexible, but it should probably be done as a separate activity and not in scope of a bugfix.

File: client/homebrew/phbStyle/phb.style.less (hunk 1 above)
Comment (calculuschild) [reply client/homebrew/phbStyle/phb.style.less:192]: @G-Ambatte, do you want to make this change and then we'll merge it?

**Review by AlexeySachkov — APPROVED** [review]

File: client/homebrew/phbStyle/phb.style.less (hunk 1 above)
Comment (G-Ambatte) [reply client/homebrew/phbStyle/phb.style.less]: I've pushed the change to `-webkit-transform` now.

**Pull request merged**